evaluations/subcoin_2_piewol.md

```diff
@@ -0,0 +1,189 @@
+
+          Running unittests src/lib.rs (target/debug/deps/subcoin_runtime_primitives-6e43d92ef6aec823)
+
+     running 0 tests
+
+     test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.00s
+
+          Running unittests src/lib.rs (target/debug/deps/subcoin_service-0b18db2ae41f5b3f)
+
+     running 1 test
+     test block_executor::tests::off_runtime_in_memory_executor_should_produce_same_result_as_runtime_disk_executor ... ok
+
+     test result: ok. 1 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in 36.71
+
+
+</details>
+
+## Docker
+Image is building. 
+
+<details>
+
+````
+ubuntu@ip-172-31-30-101:~/subcoin$ sudo docker build .
+[+] Building 1843.6s (12/12) FINISHED                            docker:default
+ => [internal] load build definition from Dockerfile                       0.1s
+ => => transferring dockerfile: 1.35kB                                     0.0s
+ => [internal] load metadata for docker.io/library/ubuntu:22.04            1.3s
+ => [internal] load .dockerignore                                          0.0s
+ => => transferring context: 2B                                            0.0s
+ => [internal] load build context                                        134.7s
+ => => transferring context: 12.66GB                                     134.6s
+ => [builder 1/5] FROM docker.io/library/ubuntu:22.04@sha256:340d9b015b19  3.5s
+ => => resolve docker.io/library/ubuntu:22.04@sha256:340d9b015b194dc6e2a1  0.0s
+ => => sha256:340d9b015b194dc6e2a13938944e0d016e57b967996 1.13kB / 1.13kB  0.0s
+ => => sha256:0eb0f877e1c869a300c442c41120e778db7161419244ee5 424B / 424B  0.0s
+ => => sha256:8a3cdc4d1ad3e314a91f76b7b99eed443f2152e3a9b 2.30kB / 2.30kB  0.0s
+ => => sha256:3713021b02770a720dea9b54c03d0ed83e03a2ef5 29.53MB / 29.53MB  1.2s
+ => => extracting sha256:3713021b02770a720dea9b54c03d0ed83e03a2ef5dce2898  2.0s
+ => [builder 2/5] WORKDIR /src                                           146.8s
+ => [builder 3/5] RUN apt-get update &&     DEBIAN_FRONTEND=noninteracti  43.5s
+ => [builder 4/5] COPY . .                                               176.3s 
+ => [builder 5/5] RUN /root/.cargo/bin/cargo build     --locked     --  1465.3s 
+ => [stage-1 2/3] COPY --from=builder /subcoin /subcoin                    0.2s 
+ => [stage-1 3/3] RUN mkdir /node-data && chown nobody:nogroup /node-data  0.2s 
+ => exporting to image                                                     0.3s 
+ => => exporting layers                                                    0.3s 
+ => => writing image sha256:9037f7d697a3625c15bc64fd85433e07bc5ebf0aebe98  0.0s
+ ````
+
+</details>
+
+## Docker
+
+The dockerfile in the delivery release didn't seem to work. The node didn't start. With your specified hash `b35da7a` I still get the error, that the disk access is causing issues. 
+
+````
+root@ip-172-31-30-101:/home/ubuntu/subcoin# docker run subv2 run
```

Comment:
If you have successfully built the image `subv2` and failed to run with this exact command locally, please try adding `--user root`:

```bash
docker run --user root subv2 run
```

Using the root is not ideal, but it's not a high priority thing, let's just bear with it for now.

Reply:
Cool, great thanks. Adding root worked. I was always under the impression that containers are always running as root. The more you know ^^.